test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1,7 +1,7 @@
 ﻿# Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# This is a Pester test suite which validate the Web cmdlets.
+# This is a Pester test suite which validates the Web cmdlets.
```

Comment:
Please revert this change. Since it is unrelated to your code change, it should be corrected in a separate PR.